grammar.js

```diff
@@ -66,6 +70,12 @@ module.exports = grammar({
       '}',
     ),
 
+    _inline_tag_false_positive: _ => prec.left(1, seq(
+      '{',
+      /[^@}]+/,
+      optional('}'),
```

Comment:
Just a note - existing queries might be matching all braces as a 'special' punctuation - since this is a false positive we probably wouldn't want to match against this

We can solve this by making it a regex, thus unqueryable, or with an alias. This also can be left as is, I'll leave that up to you, just throwing in my 2 cents

Reply:
Can `optional` take a regex? Would changing `}` to `/}/` be the only change needed?

Reply:
Pretty much, yeah, we just don't want to error out in these cases, yet would still keep it hidden from consumers. Does tokenizing this entire rule work as well? that could eliminate querying the braces too

Reply:
I just changed it to a regex. Works for me if it works for you.